modules/local/prepare_shiny.nf

```diff
@@ -11,22 +11,25 @@ process PREPARE_SHINY {
     path (app)
 
     output:
-    tuple val(meta), path ("shiny_data.csv"), emit: data
-    path ("shiny_app*")                     , emit: app
-    path ("run.sh")                         , emit: run
-    path "versions.yml"                     , emit: versions
+    tuple val(meta), path("shiny_data.csv"), emit: data
+    path "shiny_app*"                      , emit: app
+    path "static*"                         , emit: static_dir
+    path "run.sh"                          , emit: run
+    path "versions.yml"                    , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    prefix = task.ext.prefix ?: "${meta.id}"
+    def args         = task.ext.args ?: ''
+    prefix           = task.ext.prefix ?: "${meta.id}"
+    def docker_url   = "wave.seqera.io/wt/fe232c94328d/wave/build:pandas-2.1.2_pathlib-1.0.1_plotly-5.22.0_shiny-0.9.0_pruned--d898d8ae48c605ea"
```

Comment:
Would it make sense to declare it as a value input channel so that other images could be provided if desired?

Reply:
Depends on whether we want to reuse it somewhere else, I think we should discuss this with the others. Given the lack of default values in Nextflow, I would say we don't need this at the moment.

Reply:
yeah it would not be bad to have it as an input but at the same time i think this module is so custom for this specific task that i think for now we could leave it like this :) 

So for me it is approved - @JoseEspinosa you let us know if we can merge or you have strong feelings about this